src/utils/azureUtils.ts

```diff
@@ -43,14 +44,23 @@ export function createGroupConfigFromResource(resource: GenericResource, subscri
     }
 }
 
-export function getIconPath(type?: string): TreeItemIconPath {
+function getId(subscriptionId?: string, type?: string, kind?: string): string {
+    if (type?.toLowerCase() === 'microsoft.web/sites') {
+        if (kind?.toLowerCase() === 'functionapp') {
+            return `${subscriptionId}/${kind}`;
```

Comment:
I feel a little uneasy about reimagining the type as something else...can we make a supported type app that considers both type and optional kind as part of the key?